Marlin/ultralcd_impl_HD44780.h

```diff
@@ -530,13 +530,15 @@ void lcd_print(char c) { charset_mapper(c); }
       safe_delay(2000);
     #endif
 
-    lcd_implementation_clear();
+    /*
```

Comment:
marlin dev just recently changed that back as it was missing there for some time now too :) I fixed it to their exact code in "base" already.